yarn.lock

```diff
@@ -7517,11 +7517,11 @@ __metadata:
     overlayscrollbars: ^1.13.1
     polished: ^4.2.2
     prettier: ">=2.2.1 <=2.3.0"
+    prism-react-renderer: ^1.3.1
     prop-types: ^15.7.2
     qs: ^6.10.0
     react-colorful: ^5.1.2
     react-popper-tooltip: ^3.1.1
-    react-syntax-highlighter: ^15.4.5
```

Comment:
Interestingly this one didn't cause a bigger change here. Is it sill being used by other components?